app/controller/alchemy/pg_search/controller_methods.rb

```diff
@@ -83,6 +81,16 @@ def search_result_page_layout
       def paginate_per
         Alchemy::PgSearch.config[:paginate_per]
       end
+
+      private
+
+      def set_preview_query
+        if self.class == Alchemy::Admin::PagesController && params[:query].blank?
+          element = search_result_page.draft_version.elements.first
```

Comment:
We should find the element by its name. It might be that more elements are available on this page type.

```suggestion
          element = search_result_page.draft_version.elements.named(:search_results).first
```